sequelize/seeders/04-demo-player.js

```diff
@@ -1,45 +1,19 @@
+const faker = require('faker');
+
 module.exports = {
     up: async function (queryInterface, Sequelize) {
-        return queryInterface.bulkInsert('players', [{
-            firstName: 'Evgeny',
-            lastName: 'Butkevich'
-        },
-        {
-            firstName: 'Anton',
-            lastName: 'Fomenok'
-        },
-        {
-            firstName: 'Mikhail',
-            lastName: 'Sudnikovich'
-        },
-        {
-            firstName: 'Ilya',
-            lastName: 'Faley'
-        },
-        {
-            firstName: 'Arthur',
-            lastName: 'Shket'
-        },
-        {
-            firstName: 'Evgeny',
-            lastName: 'Karlenok'
-        },
-        {
-            firstName: 'Anton',
-            lastName: 'Kucher'
-        },
-        {
-            firstName: 'Ilya',
-            lastName: 'Sinko'
-        },
-        {
-            firstName: 'Evgeny',
-            lastName: 'Gorokhovich'
-        },
-        {
-            firstName: 'Vova',
-            lastName: 'Linnik'
-        }]);
+        const recordsList = [];
+        const imageUrl = 'https://i.pravatar.cc/150?img=';
+
+        for (let i = 1; i <= 50; i++) {
```

Comment:
Please move 50 and imageUrl to the top of the file and make variables capitalized

Reply:
Done